addons/project/migrations/12.0.1.1/pre-migration.py

```diff
@@ -9,6 +9,23 @@
 }
 
 
+def compute_project_task_rating_last_value(cr):
+    openupgrade.logged_query(
+        cr, """UPDATE project_task pt
+        SET rating_last_value = rr.rating
+        FROM rating_rating rr
+        WHERE rr.res_model = 'project.task' AND rr.res_id = pt.id
+        """
+    )
+
+
 @openupgrade.migrate()
 def migrate(env, version):
     openupgrade.copy_columns(env.cr, column_copies)
+    if not openupgrade.column_exists(
+            env.cr, 'project_task', 'rating_last_value'):
+        openupgrade.logged_query(
+            env.cr, """
+            ALTER TABLE project_task ADD COLUMN rating_last_value INTEGER""",
```

Comment:
The field is a float one... And why do you need to compute it again? Please always put in the comment in the analysis work for avoiding later questions.